intro.html

```diff
@@ -0,0 +1,130 @@
+      rel="stylesheet"
+    />
+    <script src="/js/analytics.js"></script>
+  </head>
+  <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item">
+          <a class="dropdown-link" href="https://my.realdevsquad.com/">
+            My Profile
+          </a>
+        </li>
+        <hr class="line" />
+        <li class="dropdown-item dropdown-link" id="signout-option">
+          Sign out
+        </li>
+      </ul>
+    </div>
+    <button id="scroll-to-top-btn">
+      <img src="./img/arrow_up.svg" alt="scroll to top button" />
+    </button>
+    <nav>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="nav-login">
+        <a
+          class="btn-login"
+          href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+        >
+          <button class="btn-login-text">
+            Sign In
+            <img
+              src="img/github.png"
+              alt="GitHub Icon"
+              height="15px"
+              width="15px"
+            />
+          </button>
+        </a>
+
+        <div class="user-greet">
+          <div class="user-greet-msg">Hello, User!</div>
+          <img class="user-profile-pic" />
+        </div>
+      </div>
+
+      <ul class="nav-menu">
+        <li class="nav-logo-li">
+          <a href="https://realdevsquad.com/">
+            <img
+              src="img/Real-Dev-Squad@1x.png"
+              alt="home nav logo"
+              height="50px"
+              width="50px"
+            />
+          </a>
+        </li>
+        <li class="home-tab"><a href="http://realdevsquad.com/">Home</a></li>
+        <li><a href="https://welcome.realdevsquad.com/">Welcome</a></li>
+        <li><a href="./events.html">Events</a></li>
+        <li><a href="https://members.realdevsquad.com/">Members</a></li>
+        <li><a href="https://crypto.realdevsquad.com/">Crypto</a></li>
+        <li><a href="https://status.realdevsquad.com/">Status</a></li>
+        <li class="nav-login-li">
+          <a
+            class="btn-login"
+            href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+          >
+            <button class="btn-login-text">
+              Sign In With GitHub
+              <img
+                src="img/github.png"
+                alt="GitHub Icon"
+                height="15px"
+                width="15px"
+              />
+            </button>
+          </a>
+          <div class="user-greet">
+            <div class="user-greet-msg">Hello, User!</div>
+            <img class="user-profile-pic" />
+          </div>
+        </li>
+      </ul>
+    </nav>
+    <h1 class="loading">Loading...</h1>
+    <div class="not-authorized-page"></div>
+    <div class="not-found-page"></div>
+    <main class="intro-main"></main>
```

Comment:
can this go in the same div, named content and we can add classes based on the js response? (Please take others' opinions on this as well)

Reply:
@sahsisunny @shmbajaj what's your opinion on this??

Reply:
Why are you making more complex, it is correct.